database/migrations/2019_10_02_131328_create_card_contents.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCardContents extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('card_contents', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->timestamps();
+            $table->string('card_title');
```

Comment:
please refer to this https://laravel.com/docs/6.x/session#driver-prerequisites

Reply:
I'm sorry, I don't understand what you mean, could you please explain it?

Reply:
it's all required in the database, so I don't think we need any modifiers to add. but, I'll add a unique modifier in card title because every card is unique